Yoast/wordpress-seo
# JS independent notification dismissal

Adding the notification dismissal button inside the HTML.
Converted all notifications to real "Yoast" notifications, which will likely get Yoast styling in the future.

Used the old dismissible fallback for < 4.2 (which didn't work because of a typo) for notification styling.

*How to test*
Dismiss a notification with JavaScript enabled.
* the page should not be refreshed
* the notification should be gone on next refresh

Dismiss a notification with JavaScript disabled.
* the dismissal button should be inside the notification
* the page should refresh with nonce and dismissal information in the query string
* the notification should be gone from now on

Testing help:
Disabling the cookie sync helps testing a lot; just removing the `wpseo_seen_about_version` from the `wp_usermeta` makes the notification popup again.

`WPSEO_Admin_Init` line 60 `$this->sync_about_version_from_cookie();`

Fixes #4492
